### PR TITLE
Repair: Diverged Config

### LINKED ISSUE #334: [8pt] Repair: Resolve different configs situation
Sometimes cluster-wide configuration files are different for instances on the one machine. We have to handle this situation correctly.

### PR DESCRIPTION
**Pull request by dokshina**
- [x] Tests
- [ ] Changelog
- [ ] Documentation

Closes #334

### CODE REVIEW

File: cli/repair/app_configs.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+	for hash := range appConfigs.confByHash {
+		appConfigs.hashes = append(appConfigs.hashes, hash)
+	}
+	sort.Sort(sort.StringSlice(appConfigs.hashes))
+
+	return appConfigs, nil
+}
+
+func (d *AppConfigs) Different() bool {
```
Comment (olegrok) [inline cli/repair/app_configs.go:68]: IsDifferent?

Comment (dokshina) [reply cli/repair/app_configs.go:68]: You mean `AreDifferent`?

Comment (olegrok) [reply cli/repair/app_configs.go:68]: Yes

File: cli/repair/app_configs.go [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+		if _, found := appConfigs.confByHash[hash]; !found {
+			if appConfigs.confByHash[hash], err = getTopologyConf(topologyConfPath); err != nil {
+				return appConfigs, fmt.Errorf("Failed to get topology config: %s", err)
+			}
+		}
+
+	}
+
+	for _, insstanceIDs := range appConfigs.instancesByHash {
```
Comment (olegrok) [inline cli/repair/app_configs.go:56]: `insstanceIDs ` -> `instanceIDs `

**Review by olegrok — APPROVED** [review]

**Pull request merged**